CMakeLists.txt

```
@@ -2,6 +2,9 @@ cmake_minimum_required(VERSION 2.8...3.5 FATAL_ERROR)
 
 project(isle CXX)
 
+link_directories("$ENV{LIB}")
+include_directories("$ENV{INCLUDE}")
+
 option(ISLE_BUILD_APP "Build ISLE.EXE application" ON)
```

Add this to a cmake toolchain.

And even so, when you rerun cmake without these environment variables set, nothing will be added to the include and library search directories.
```suggestion
project(isle CXX)

option(ISLE_BUILD_APP "Build ISLE.EXE application" ON)
```